packages/synapse-react-client/src/components/widgets/Range.tsx

```diff
@@ -72,39 +74,85 @@ export function Range(props: RangeProps) {
   }
 
   return (
-    <div className={className} style={{ display: 'flex', flexWrap: 'wrap' }}>
-      <div style={{ marginRight: '10px' }}>
-        <input
-          aria-label="min"
-          key="range_min"
-          type={props.type}
```

Comment:
The old range picker supports different types.  We cannot (should not) use a DatePicker for all types, only dates.  In other cases, should probably use a MUI TextField for styling (with the type property set)

Reply:
@jay-hodgson just pushed an update to address this problem. Can you take a look?

Reply:
![Screenshot 2025-02-26 at 3 07 40 PM](https://github.com/user-attachments/assets/4b86e62a-78e6-47bd-8695-b34eebc6ed54)